commons/constants.py

```diff
@@ -58,6 +58,7 @@
 NAMESPACE = "default"
 CONTROL_POD_NAME_PREFIX = "cortx-control"
 HA_SHUTDOWN_SIGNAL_PATH = "/root/cortx-test/scripts/server_scripts/ha_shutdown_signal.py"
+MOCK_MONITOR_PATH = "/cortx-ha/ha/util/health_generator/mock_health_event_publisher.py"
```

Comment:
so we would need to get from repo only and can't create one in our scripts?

Reply:
This script has some references from ha repo so for now we have to get it from repo. But as per Madhura from HA team, we will get RPM for the same. Once we get RPM, this thing won't be needed. We can just install RPM inside HA pod and simulate failures.